packages/alea-frontend/pages/api/get-students-enrolled-in-course.ts

```diff
@@ -1,23 +1,67 @@
+
+import { Action, CURRENT_TERM, getResourceId, ResourceName } from '@stex-react/utils';
 import { NextApiRequest, NextApiResponse } from 'next';
-import { executeAndEndSet500OnError } from './comment-utils';
 import { getCourseEnrollmentAcl } from '../course-home/[courseId]';
+import { getCacheKey } from './acl-utils/acl-common-utils';
+import { returnAclIdForResourceIdAndActionId } from './acl-utils/resourceaccess-utils/resource-common-utils';
+
+const courseIds = ['ai-1', 'ai-2', 'lbs', 'iwgs-1', 'iwgs-2'];
+
+const instructorResourceAndAction = [
+  { resource: ResourceName.COURSE_NOTES, action: Action.MUTATE },
+  { resource: ResourceName.COURSE_QUIZ, action: Action.MUTATE },
+  { resource: ResourceName.COURSE_STUDY_BUDDY, action: Action.MODERATE },
+  { resource: ResourceName.COURSE_HOMEWORK, action: Action.MUTATE },
+  { resource: ResourceName.COURSE_ACCESS, action: Action.ACCESS_CONTROL },
+];
+
 
 export default async function handler(req: NextApiRequest, res: NextApiResponse) {
   try {
     const { courseId, instanceId } = req.query;
     if (!courseId || !instanceId) {
       return res.status(400).json({ error: 'Missing courseId or instanceId in query parameters.' });
     }
+
+    const instructorResourceIdAndAction = courseIds.flatMap((course) =>
+      instructorResourceAndAction.map(({ resource, action }) => ({
+        resourceId: getResourceId(resource, {
+          courseId: course,
+          instanceId: CURRENT_TERM,
+        }),
+        action,
+      }))
+    );
+
     const aclId = getCourseEnrollmentAcl(courseId as string, instanceId as string);
-    const students = await executeAndEndSet500OnError(
-      'SELECT memberUserId FROM ACLMembership WHERE parentACLId = ?',
-      [aclId],
-      res
+    const enrolledPersons = await CACHE_STORE.getFromSet(getCacheKey(aclId));
```

Comment:
As mentioned earlier, create a helper function (`getAllAclMembers`) in the appropriate location (`acl-common-utils.ts`) and use its everywhere instead of copy paste